pr2eus/pr2-interface.l

```diff
@@ -407,7 +407,7 @@ Example: (send self :gripper :rarm :position) => 0.00"
 
 (defmethod pr2-interface
   (:angle-vector-with-constraint
-   (av1 &optional (tm 3000) (arm :arms) &key (rotation-axis t) (translation-axis t) &rest args)
+   (av1 &optional (tm 3000) (arm :arms) &key (rotation-axis t) (translation-axis t) &allow-other-keys)
```

Comment:
1. `&allow-other-keys` only works with `&rest`, so it is wrong to delete `&rest`.
1. Currently variable `args` is not used anywhere, but this function works so I think we don't have to change if you don't have any trouble on this.